content/docs/studio/visualize-experiments.md

```diff
@@ -0,0 +1,40 @@
+# Visualize and Compare Experiments
+
+You can visualize and compare experiements using a variety of tools.
+
+## Generate metric plots
+
+You can visualize certain metrics of machine learning experiments as plots.
+Usual plot examples are AUC curves, loss functions, and confusion matrices,
+among others. This type of metrics files are created by users, or generated by
+user data processing code, and can be defined in dvc.yaml (plots field) for
+tracking (optional). Refer to the
+[DVC plots documentation](/doc/command-reference/plots) for details on how plots
+are generated.
+
+To generate metric plots, select one or more experiements (represented by the
+commits), and click on the 'Show plots' button.
+
+<img src="/img/studio/select_commits.png" alt="drawing" width="300"/>
+
+The plots will appear below the tabular display. If you have selected more than
+one experiment, you can use the plots to compare them.
+
+<img src="/img/studio/plots.png" alt="drawing" width="300"/>
+
+## Generate trend charts
+
+Click on the Trends button to generate a plot of how the metrics changed over
+the course of the different experiments. For each metric, the trend charts show
```

Comment:
Can we make the first sentence into a p and insert an image of the `Trends` button? It's not easy to find by just looking at the view (I had to Ctrl+F the first time). Something like this:

![image](https://user-images.githubusercontent.com/1477535/117914676-6cd44580-b2a9-11eb-8362-5107e92a7a0e.png)

Reply:
Maybe also for `Compare` for consistency. That button is much more obvious/visible though (so maybe this is a deisgn issue in fact).

Reply:
Added to new ticket https://github.com/iterative/dvc.org/issues/2468